File: app/src/main/java/dev/ragnarok/fenrir/api/model/upload/UploadAudioDto.java
package dev.ragnarok.fenrir.api.model.upload;

import androidx.annotation.NonNull;
import androidx.annotation.Nullable;

import com.google.gson.annotations.SerializedName;

public class UploadAudioDto {

    @Nullable
    @SerializedName("server")
    public String server;

    @Nullable
    @SerializedName("audio")
    public String audio;

    @Nullable
    @SerializedName("hash")
    public String hash;

    @NonNull
    @Override
    public String toString() {
        return "UploadAudioDto{" +
                "server='" + server + '\'' +
                ", audio='" + audio + '\'' +
                ", hash='" + hash + '\'' +
                '}';
    }
}

File: app/src/main/kotlin/dev/ragnarok/fenrir/api/model/upload/UploadAudioDto.kt
package dev.ragnarok.fenrir.api.model.upload

import com.google.gson.annotations.SerializedName

class UploadAudioDto {
    @SerializedName("server")
    var server: String? = null

    @SerializedName("audio")
    var audio: String? = null

    @SerializedName("hash")
    var hash: String? = null
    override fun toString(): String {
        return "UploadAudioDto{" +
                "server='" + server + '\'' +
                ", audio='" + audio + '\'' +
                ", hash='" + hash + '\'' +
                '}'
    }
}